frontend/public/components/routes.tsx

```diff
@@ -230,6 +241,35 @@ const RouteDetails: React.SFC<RoutesDetailsProps> = ({obj: route}) => <React.Fra
     <h1 className="co-section-title">TLS Settings</h1>
     <TLSSettings tls={route.spec.tls} />
   </div>
+  { !_.isEmpty(route.spec.alternateBackends) && <div className="co-m-pane__body">
+    <h1 className="co-section-title">Traffic</h1>
+    <p className="co-m-pane__explanation">
+      This route splits traffic across multiple services.
+    </p>
+    <div className="co-table-container">
+      <table className="table">
+        <thead>
+          <tr>
+            <th>Service</th>
+            <th>Weight</th>
+            <th>Percent</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td><ResourceLink kind={route.spec.to.kind} name={route.spec.to.name} namespace={route.metadata.namespace} title={route.spec.to.name} /></td>
+            <td>{route.spec.to.weight}</td>
+            <td>{calcTrafficPercentage(route.spec.to.weight, route)}%</td>
+          </tr>
+          {_.map(route.spec.alternateBackends, (alternate, i) => <tr key={i}>
+            <td><ResourceLink kind={alternate.kind} name={alternate.name} namespace={route.metadata.namespace} title={alternate.name} /></td>
+            <td>{alternate.weight}</td>
+            <td>{calcTrafficPercentage(alternate.weight, route)}%</td>
+          </tr>)}
```

Comment:
I'd create a component to avoid repeating the same lines more than once.

```jsx
const RouteTargetRow = ({route, target}) => <tr>
  <td><ResourceLink kind={target.kind} name={target.name} namespace={route.metadata.namespace} title={target.name} /></td>
  <td>{target.weight}</td>
  <td>{calcTrafficPercentage(target.weight, route)}%</td>
</tr>;

// ...

<tbody>
  <RouteTargetRow route={route} target={route.spec.to} />
  {_.map(route.spec.alternateBackends, (alternate, i) => <RouteTargetRow key={i} route={route} target={alternate} />)}
</tbody>
```

Reply:
Good idea, thanks!

Reply:
Added